src/alert-manager/src/index.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+/**
+ * Implementation of Alert Handler.
+ * Init and start server instance.
+ */
+
+const express = require('express');
+const unirest = require('unirest');
+const bearerToken = require('express-bearer-token');
+
+const app = express();
+
+app.use(express.json());
+app.use(bearerToken());
+
+app.post('/alert-handler', (req, res) => {
```

Comment:
since we may have further more than one actions handled by the webhook, I think you'd better clearly specify the actions like `/alert-handler/stop-job`, `/alert-handler/tag-job` and so on

Reply:
> since we may have further more than one actions handled by the webhook, I think you'd better clearly specify the actions like `/alert-handler/stop-job`, `/alert-handler/tag-job` and so on

Yes, I'll fix that after refining the doc.